### PR TITLE
Remove string literal to string conversion

### PR DESCRIPTION
**Pull request by daria-cara**
Remove instances of unnecessary string literal-to-string conversion such as `str("some_literal")`. This conversion was probably left over after import of `unicode_literals` was removed when `astropy` stopped supporting Python 2.7. In Python 3 this conversion is no longer needed.

### CODE REVIEW

File: astropy/tests/plugins/display.py [hunk 1]
```diff
@@ -3,8 +3,6 @@
 This plugin provides customization of the header displayed by pytest for
 reporting purposes.
 """
-from __future__ import (absolute_import, division, print_function,
```
Comment (bsipocz) [inline astropy/tests/plugins/display.py]: wow, not sure how this survived here so long.

**Comment by bsipocz** [comment]
Looks all perfectly good and uncontroversial to me.  The docs failure is totally unrelated, and since I didn't spot a single line that has any effect on the generated docs, with two approvals I go ahead and merge.

**Comment by bsipocz** [comment]
Thank you @daria-cara!

File: astropy/tests/plugins/display.py (hunk 1 above)
Comment (pllim) [reply astropy/tests/plugins/display.py]: I had the exact same thought!

**Pull request merged**